bin/gobuild.sh

```diff
@@ -59,5 +60,5 @@ while read line; do
 done < "${BUILDINFO}"
 
 # forgoing -i (incremental build) because it will be deprecated by tool chain. 
-time GOOS=${GOOS} GOARCH=${GOARCH} ${GOBINARY} build ${V} ${GOBUILDFLAGS} -o ${OUT} \
+time GOOS=${GOOS} GOARCH=${GOARCH} ${GOBINARY} build ${V} ${GOBUILDFLAGS} -gcflags "${GCFLAGS}" -o ${OUT} \
```

Comment:
I think you could use this form to modify the build line only if GCFLAGS is set: 

${GCFLAGS:+-gcflags "$GCFLAGS"}

Reply:
good suggestion